cluster.go

```diff
@@ -748,8 +792,14 @@ process:
 		goto process
 	case RedirectRetry:
 		if c.retry {
-			runtime.Gosched()
-			goto retry
+			shouldRetry := c.retryHandler.WaitUntilNextRetry(ctx, attempts, resp.Error())
+			if shouldRetry {
+				attempts++
+				goto retry
+			}
+			if err != nil {
+				return newErrResult(err)
+			}
```

Comment:
Could this be removed?

Reply:
[1725eb5](https://github.com/redis/rueidis/pull/643/commits/1725eb5ca04b3c745838debe0cfa555fa2c2e227)